src/types/config.ts

```diff
@@ -5,11 +5,17 @@ import { ContractDeploymentBlock } from 'api/exchange/ExchangeApi'
 import { Network } from 'types'
 import { TokenDetailsConfigLegacy } from '@gnosis.pm/dex-js'
 
-export interface TokenSelection {
+interface BaseTokenSelection {
   sellToken: string
   receiveToken: string
 }
 
+export interface TokenSelection extends BaseTokenSelection {
+  network: {
+    [n: string]: BaseTokenSelection | undefined
+  }
+}
+
```

Comment:
If you want more specific types, can do
```ts
export type TokenSelection = BaseTokenSelection & {
  network: {
    [n in Network]?: BaseTokenSelection
  }
}
```

can't do the same with interfaces though